frontend/pages/SoftwarePage/SoftwareOSDetailsPage/SoftwareOSDetailsPage.tsx

```diff
@@ -95,7 +140,11 @@ const SoftwareOSDetailsPage = ({
       return <Spinner />;
     }
 
-    if (isError) {
+    if (isOsVersionError) {
+      // confirm okay to cast to AxiosError like this
```

Comment:
can you do me the favor of removing all of these comments, including the original one I left 🙂

Reply:
@jacobshandling it's actually broken, I'm going to refactor after standup to use `useErrorHandler` I found

Reply:
oh!